runtime/src/lib.rs

```diff
@@ -813,6 +815,17 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl merkle::MerkleApi<Block> for Runtime {
+		fn get_leaves(tree_id: u32, index: u32) -> Option<ScalarData> {
```

Comment:
Should we call this `get_leaf`?
```suggestion
		fn get_leaf(tree_id: u32, index: u32) -> Option<ScalarData> {
```